src/EnergyPlus/AirLoopHVACDOAS.cc

```diff
@@ -753,8 +753,8 @@ namespace AirLoopHVACDOAS {
                 if (AirLoopNames != fields.end()) {
                     auto AirLoopArray = AirLoopNames.value();
                     int num = 0;
-                    for (auto AirLoopHAVCName : AirLoopArray) {
-                        std::string name = AirLoopHAVCName.at("airloophvac_name");
+                    for (auto AirLoopHVACName : AirLoopArray) {
+                        std::string name = UtilityRoutines::MakeUPPERCase(AirLoopHVACName.at("airloophvac_name"));
```

Comment:
Fix is here. (I also fixed an unrelated typo in variable name)

Reply:
Makes perfect sense.  And thanks for the typo fix.

Reply:
@Myoldmopar This *doesn't* make perfect sense. It shouldn't be necessary to `MakeUPPERCase` object names during input processing. We don't do that everywhere. Shouldn't `FindItemInList` be case-insensitive as a general solution to idfs that may have varied case for the same name? In the olden days, the input processor pushed all strings to UPPERcase so it didn't matter, but that's not been happening since the new IP with epJSON. Have we just been lucky that most files use the same case for all references to a name? My question here is prompted by a new helpdesk ticket with a similar issue for ZoneHVAC:HybridUnitaryHVAC, see #8614.

Reply:
There is extensive examples of getinput routines that do makeuppercase already, and it seems it really is needed. I agree that this logic should probably be baked in to avoid having to worry about it (we support case insensitiveness, then act like it). If a modification is done at a higher level (inputprocessor or the finditeminlist) then existing calls to makeupercase prior to calling routines need to be eliminated to avoid the overhead of doing it twice though